src/Console/Command/LdapSyncCommand.php

```diff
@@ -158,6 +159,23 @@ private function findUsers($dn): Generator
         }
     }
 
+    /**
+     * Find a valid email
+     *
+     * @param string[] $emails
+     * @return string|null
+     */
+    private function findValidEmail(array $emails): ?string
```

Comment:
you're not using the value, so rename the method to `hasValidEmail(array $emails): bool`